marta/api.py

```diff
@@ -41,8 +41,12 @@ def get_trains(line=None, station=None, destination=None, api_key=None):
     :param api_key (str): API key to override environment variable
     :return: list of Train objects
     """
-    response = requests.get('{}{}?apikey={}'.format(_BASE_URL, _TRAIN_PATH, api_key))
-    data = loads(response.text)
+    endpoint = '{}{}?apikey={}'.format(_BASE_URL, _TRAIN_PATH, api_key)
+    response = requests.get(endpoint)
+    try:
```

Comment:
Instead of depending on `loads()` failing, I think it would be better if we check `response.status_code` here and raise the `APIKeyError` if it's a 403 (or whatever code the API returns in that case).

Reply:
It looks like they send a 401 response, do you think it'd make more sense to check equality for 401 or inequality on 200? I'm personally leaning towards inequality on 200, in case they later change the response from 401 to 403, but that also has the weakness of failing on a 500 response.

Of course, since the error message suggests visiting the url in-browser anyway, I'm not sure that would be a huge deal.